moveit_py/src/moveit/moveit_ros/moveit_cpp/moveit_cpp.cpp

```diff
@@ -62,7 +64,7 @@ void initMoveitPy(py::module& m)
 
       .def(py::init([](const std::string& node_name, const std::string& name_space,
                        const std::vector<std::string>& launch_params_filepaths, const py::object& config_dict,
-                       bool provide_planning_service) {
+                       bool provide_planning_service, const py::object& remappings) {
```

Comment:
Just use a std::map as argument here. This simplifies casting code below.
Wrapping the map into `std::optional` [allows to handle None values](https://github.com/pybind/pybind11/issues/1953#issuecomment-683409703).
```suggestion
                       bool provide_planning_service, const std::optional<std::map<std::string, std::string>>& remappings) {
```